app/templates/app/questions.html

```diff
@@ -0,0 +1,46 @@
+{% extends 'app/_base.html' %}
+{% load static %}
+
+
+    {% block title %}
+        Questions
+    {% endblock %}
+
+    {% block content %}
+<!--could put if user.is_authenticated here possibly -->
+    <div id="questionPage" class="container">
+        <h1 class="text-muted">Questions</h1>
+        <br>
+            <h2 name="question" style="color: #003366" id="question">[[ questionPackage.question ]]</h2>
```

Comment:
Do we need to define an inline style here or can we put it in the css or use Vue style bindings (:style)